nutrition_labels/grant_tagger.py

```diff
@@ -131,8 +103,8 @@ def evaluate(self, X, y, print_results=True):
         scores = {
             'accuracy': accuracy_score(y, y_predict),
             'f1': f1_score(y, y_predict),
-            'precision_score': precision_score(y, y_predict),
-            'recall_score': recall_score(y, y_predict)}
+            'precision_score': precision_score(y, y_predict, zero_division=0),
+            'recall_score': recall_score(y, y_predict, zero_division=0)}
```

Comment:
I take it this needed to be done again here because this PR was started before the previous PR was merged?

Reply:
yep